docs/architecture.md

```diff
@@ -21,25 +21,25 @@ connections to downstream clusters. This means managed clusters can run in priva
 NATs. The only requirement is the cluster agent needs to be able to communicate with the
 Kubernetes API of the cluster running the Fleet manager. The one exception to this is if you use
 the [manager initiated](./manager-initiated.md) cluster registration flow.  This is not required, but
-an optional patter.
```

Comment:
This is fixed by: https://github.com/rancher/fleet/pull/229
We may want to wait for that PR to be merged, and then rebase.

Reply:
Done in https://github.com/rancher/fleet/pull/227/commits/e4c539903c7c4c9f3c053705ac8b4eee63e723a8.